src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -81,7 +86,8 @@ const ChatItemMyTextV2 = ({
       message,
       messageId: data?.id,
       chatType,
-      messageType: 'regular'
+      messageType: 'regular',
+      attachments
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `attachments` in the object passed to `onContextMenuPressed` is necessary to provide context menu actions for attachments.
```diff
+       attachments
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->